src/ui/molecules/stripe-elements.svelte

```diff
@@ -79,10 +77,12 @@
       viewport = "desktop";
     }
 
+    // These values are fixed since the behaviors in stripe are rather unpredictable and
+    // the sizes of fonts and spacing expected to be the same in both viewports
```

Comment:
I simplified here given it does not work as expected, we may as well be explicit.

Reply:
Curiosity, what was the problem? I see that bodyBase was resolving to `16px` and we wanted `14px` ?